nautobot_golden_config/templatetags/gc_helpers.py

```diff
@@ -0,0 +1,27 @@
+"""Helpers for UI component rendering that extends what Nautobot Core provides."""
+
+from django import template
+from django.utils.html import format_html
+from django_jinja import library
+from nautobot.core.templatetags import helpers as core_helpers
+
+register = template.Library()
+
+
+@library.filter()
```

Comment:
These don't need to be filters. I would either look to add these to core, or just make them functions. We won't use them elsewhere for the foreseeable future.

Reply:
Don't quite understand this comment yet, but will re-review the UI component options. I thought this was the only way to use the value_transform